### PR TITLE
add masterCA to serviceAccountConfig

### PR DESCRIPTION
**Pull request by deads2k**
https://github.com/openshift/origin/pull/3622 adds a `masterCA` for service accounts.

@liggitt

### CODE REVIEW

File: admin_guide/master_node_configuration.adoc [hunk 1]
```diff
@@ -231,6 +231,7 @@ serviceAccountConfig:
   - default
   - builder
   - deployer
+  masterCA: openshift.local.config/master/ca.crt
```
Comment (liggitt) [inline admin_guide/master_node_configuration.adoc]: `ca.crt`

File: admin_guide/service_accounts.adoc [hunk 2]
```diff
@@ -59,13 +59,15 @@ to specify a `privateKeyFile` (for signing), and a matching public key file in t
 ----
 serviceAccountConfig:
   ...
-  privateKeyFile: serviceaccounts.private.key <1>
+  masterCA: openshift.local.config/master/ca.crt <1>
```
Comment (liggitt) [inline admin_guide/service_accounts.adoc]: `ca.crt`

Comment (deads2k) [inline admin_guide/service_accounts.adoc]: ah, bug in the origin pull.  Need to add to file paths.

File: dev_guide/service_accounts.adoc [hunk 3]
```diff
@@ -155,7 +155,7 @@ This example calls the *_users/~_* API to get information about the user identif
 ----
 $ TOKEN="$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
 
-$ curl -k "https://openshift.default.svc.cluster.local/oapi/v1/users/~" \
+$ curl --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt "https://openshift.default.svc.cluster.local/oapi/v1/users/~" \
```
Comment (liggitt) [inline dev_guide/service_accounts.adoc]: put on a line by itself for readability?
```
$ curl "https://openshift.default.svc.cluster.local/oapi/v1/users/~" \
  --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
  -H "Authorization: Bearer $TOKEN"
```

**Comment by deads2k** [comment]
comments addressed.

**Comment by liggitt** [comment]
LGTM

**Comment by deads2k** [comment]
squashed.

**Comment by adellape** [comment]
@deads2k Thanks. So I see this is merged in Origin, and assume will make its way into OSE in 3.0.0.x, or 3.0.1?

Merging, and will follow-up with any conditionalizing for Origin vs OSE.

**Comment by liggitt** [comment]
will be in 3.0.1

**Pull request merged**